runner/seccomp_linux.go

```
@@ -0,0 +1,205 @@
+//go:build linux
+
+package runner
+
+import (
+	"errors"
+	"fmt"
+	"runtime"
+	"syscall"
+	"unsafe"
+
+	"github.com/hustoj/runner/sec"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(goimports)